### PR TITLE
Krisp latest version support

### PR DESCRIPTION
**Pull request by aashot**
This pull request updates our Krisp integration to support the latest version of the Krisp SDK, ensuring compatibility with recent improvements and optimizations.

Commit: [40c5de7](https://github.com/jitsi/jitsi-meet/commit/40c5de750f210e6f5bf3fc8946f6a61ab248aa9d)
Author: Ashot Hovhannisyan
Date: 2024-11-05
Message: feat(noise-suppression): update Krisp setup to use the latest SDK version

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -209,14 +209,45 @@ var config = {
...
+    //   - https://meet.example.com/libs/krisp/models/model_inbound_8.kw
+    //   - https://meet.example.com/libs/krisp/models/model_inbound_16.kw
+    //     In case when you want to use inbound noise suppression models
+    // NOTE: Krisp JS SDK v2.0.0 was tested.
     // noiseSuppression: {
     //     krisp: {
-    //         enabled: false,
-    //         logProcessStats: false,
-    //         debugLogs: false,
+    //         enabled: true,
```
Comment (saghul) [inline config.js:224]: Please show the default options in the comment.

**Comment by saghul** [comment]
Looks good!

**Comment by saghul** [comment]
Jenkins test this please.

**Comment by aashot** [comment]
Hi @saghul!
Is there anything required from my side to move forward with merging this feature?

**Comment by saghul** [comment]
Merged, cheers!

**Pull request merged**